docs/man_bodhi.rst

```diff
@@ -243,6 +243,58 @@ The ``updates`` command allows users to interact with bodhi updates.
 
     Request that the given update be changed to the given state. ``update`` should be given by
     update id, and ``state`` should be one of testing, stable, unpush, obsolete, or revoke.
+    
+    
+    
```

Comment:
Small nitpick, but can you change this to two spaces instead of 3, similar to what PEP-8 would want?